storage/zynq7000-flash/flashdrv.c

```diff
@@ -566,15 +566,15 @@ static ssize_t flashdrv_mtdWrite(struct _storage_t *strg, off_t offs, const void
 
 	while (tempSz < len) {
 		chunkSz = ((len - tempSz) > strg->dev->mtd->writeBuffsz) ? strg->dev->mtd->writeBuffsz : (len - tempSz);
-		res = _flashdrv_pageProgram(strg->dev->id, strg->start + offs + tempSz, (const char *)data + tempSz, chunkSz);
+		res = _flashdrv_pageProgram(strg->dev->ctx->id, offs + tempSz, (const char *)data + tempSz, chunkSz);
```

Comment:
Removed `strg->start` without corresponding a change in `_flashdrv_pageProgram`. Is this intended?